### DISCUSSION TITLE
Please add this important features

### DISCUSSION THREAD

**Discussion by p82183645** [Zrips/CMI#2375] [Feature Requests]
Use commands to consume money

like
/cmi back , consume 50$

**Comment by kassadGLA** [comment]
You can use custom aliases (CustomAlias.yml) with [Specialized Command](http://www.zrips.net/cmi/commands/specialized/)
```
CustomAlias:
  back:
  - moneycost:50! cmi back
```
Note. Before use it set to `false` alias `back` in Alias.yml:
```
Alias:
  /back: false
```

**Comment by p82183645** [comment]
very thank
